### PR TITLE
Implement stack top frame filtering

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -248,9 +248,18 @@ CallsiteRecord.fromStackFrames = function (stackFrames, fnName, typeName) {
     return null;
 };
 
-CallsiteRecord.fromError = function (error) {
+CallsiteRecord.fromError = function (error, topFrameFilter) {
```
Comment (inikulin) [inline lib/index.js]: Maybe `findTopFrame`?

Comment (inikulin) [inline lib/index.js]: Or it's better `isTopFrame`

**Comment by inikulin** [comment]
Can you add info to README as well, please?

File: lib/index.js (hunk 1 above)
Comment (VasilyStrelyaev) [inline lib/index.js]: such a mess with this name...

"Okay, so I have a stack. But I can immediately say which frame is on top. Why am I suppose to provide the `isTopFrame` callback?"

In this sense, `findTopFrame` or even `selectTopFrame` look better. But they are more like method names...

Comment (VasilyStrelyaev) [inline lib/index.js]: A name with `...filter...` is even better but I can't come up with a good one

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**